README.md

```diff
@@ -32,6 +32,7 @@ How does it work?
 * `Array.reverse` should not be used misleadingly ([`no-misleading-array-reverse`]) ([`requires type-check`])
 * Non-existent operators '=+', '=-' and '=!' should not be used ([`no-misspelled-operator`])
 * Multiline string literals should not be used ([`no-multiline-string-literals`])
+* Redundant pairs of parentheses should be removed ([`no-redundant-parenthese`])
```

Comment:
@vilchik-elena typo in link `no-redundant-parentheses`

Reply:
I knew I will do it :) thanks